Makefile.calico-node

```diff
@@ -340,7 +345,8 @@ node-ut:
 
 PHONY: node-test-containerized
 ## Run the tests in a container. Useful for CI, Mac dev.
-node-test-containerized:
+node-test-containerized: vendor run-etcd-host
 	docker run --rm -v $(CURDIR):/go/src/$(PACKAGE_NAME):rw \
-    -e LOCAL_USER_ID=$(LOCAL_USER_ID) \
-    $(CALICO_BUILD) sh -c 'cd /go/src/$(PACKAGE_NAME) && make node-ut'
+	-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+	--net=host \
```

Comment:
I checked out libnetwork and cni plugins and they both use `--net=host` for containerized testing.  So I'm not sure what I can do to remove that. If you want to work through it with me I'm game.